polyfills/StyleSheetList/@@iterator/config.json

```diff
@@ -0,0 +1,20 @@
+{
+	"browsers": {
+		"ie": "8 <",
+		"firefox": "*",
+		"chrome": "*",
+		"safari": "*",
+		"android": "*",
+		"ios_saf": "*",
+		"opera": "*"
+	},
+	"dependencies": [
+		"_ArrayIterator",
+		"Symbol.iterator"
```

Comment:
No it doesn't, IE8 needs a `StyleSheetList` polyfill. I decided to not target IE8 for now.